opentreemap/treemap/templatetags/form_extras.py

```diff
@@ -288,8 +288,11 @@ def field_type_label_choices(model, field_name, label=None,
         field_type = udf_dict['type']
         label = label if label else udf_field_name
         if 'choices' in udf_dict:
+            choices = udf_dict['choices']
+            protected_choices = udf_dict.get('protected_choices', [])
+            all_choices = protected_choices + choices
```

Comment:
Refactored this to retrieve the UDF field directly instead of the dict and it seems to work.